### PR TITLE
fixing PSSA warnings 

### PR DESCRIPTION
**Pull request by mbreakey3**
- mainly switching null comparisons and deleting unused variables

---
This change is [Reviewable](https://reviewable.io/reviews/powershell/xwebadministration/141)

### CODE REVIEW

**Comment by mbreakey3** [comment]
There was also:
 - An unapproved verb usage (Modify) that I changed to 'Edit'
- '?' used instead of 'Where-Object'
- Incorrect return-output types
- Write-Host used instead of Write-Verbose

File: DSCResources/MSFT_xIisModule/MSFT_xIisModule.psm1 [hunk 1]
```diff
@@ -65,7 +65,7 @@ function Get-IisHandler
     )
 
     Trace-Message "Getting Handler for $Name in Site $SiteName"
-    return get-webconfiguration -Filter 'System.WebServer/handlers/*' -PSPath (Get-IisSitePath -SiteName $SiteName) | ?{$_.Name -ieq $Name}
+    return get-webconfiguration -Filter 'System.WebServer/handlers/*' -PSPath (Get-IisSitePath -SiteName $SiteName) | Where-Object{$_.Name -ieq $Name}
```
Comment (PlagueHO) [inline DSCResources/MSFT_xIisModule/MSFT_xIisModule.psm1]: 100 char line limit hit here - can we break the line up with back ticks?

File: DSCResources/MSFT_xIisModule/MSFT_xIisModule.psm1 [hunk 2]
```diff
@@ -148,7 +147,7 @@ function Get-TargetResource
         # bug(TBD) deal with this better, maybe a seperate resource....
         If($handler.Modules -eq 'FastCgiModule')
         {
-            $fastCgi = Get-WebConfiguration /system.webServer/fastCgi/* -PSPath (Get-IisSitePath -SiteName $SiteName) | ?{$_.FullPath -ieq $handler.ScriptProcessor}
+            $fastCgi = Get-WebConfiguration /system.webServer/fastCgi/* -PSPath (Get-IisSitePath -SiteName $SiteName) | Where-Object{$_.FullPath -ieq $handler.ScriptProcessor}
```
Comment (PlagueHO) [inline DSCResources/MSFT_xIisModule/MSFT_xIisModule.psm1]: 100 char line limit hit here - can we break the line up with back ticks?

File: DSCResources/MSFT_xIisModule/xIisModuleDesigner.ps1 [hunk 3]
```diff
@@ -30,7 +29,7 @@ $resources += New-xDscResourceProperty -Name EndPointSetup -Type Boolean -Attrib
 
 
 
-Write-Host updating...
+Write-Verbose updating...
```
Comment (PlagueHO) [inline DSCResources/MSFT_xIisModule/xIisModuleDesigner.ps1]: I know not strictly necessary and all, but could we wrap the updating... in quotes - just sort of feels wrong :smile:

File: DSCResources/MSFT_xWebConfigKeyValue/MSFT_xWebConfigKeyValue.psm1 [hunk 4]
```diff
@@ -85,7 +85,7 @@ function Set-TargetResource
             {
                 $propertyName = $Key
             }
-            Modify-Item -propertyName $propertyName -oldValue $existingvalue -newValue $Value -isAttribute $IsAttribute -websitePath $WebsitePath -configSection $ConfigSection
+            Edit-Item -propertyName $propertyName -oldValue $existingvalue -newValue $Value -isAttribute $IsAttribute -websitePath $WebsitePath -configSection $ConfigSection
```
Comment (PlagueHO) [inline DSCResources/MSFT_xWebConfigKeyValue/MSFT_xWebConfigKeyValue.psm1]: 100 char line limit hit here - can we break the line up with back ticks?

File: DSCResources/MSFT_xWebConfigKeyValue/MSFT_xWebConfigKeyValue.psm1 [hunk 5]
```diff
@@ -200,7 +200,7 @@ function Add-item([string]$key, [string]$value, [Boolean]$isAttribute, [string]$
     }
 }
 
-function Modify-Item([string]$propertyName, [string]$oldValue, [string]$newValue, [Boolean]$isAttribute, [string]$websitePath, [string]$configSection)
+function Edit-Item([string]$propertyName, [string]$oldValue, [string]$newValue, [Boolean]$isAttribute, [string]$websitePath, [string]$configSection)
```
Comment (PlagueHO) [inline DSCResources/MSFT_xWebConfigKeyValue/MSFT_xWebConfigKeyValue.psm1]: 100 char line limit hit here - can we break the line up with back ticks?

**Comment by mbreakey3** [comment]
Are there any further comments before I merge these changes?

**Comment by TravisEz13** [comment]
Reviewed 6 of 9 files at r1, 15 of 15 files at r2, 3 of 3 files at r3.
Review status: all files reviewed at latest revision, 5 unresolved discussions.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/powershell/xwebadministration/141)*

**Comment by mbreakey3** [comment]
@TravisEz13 - any updates or comments on these changes?

**Comment by TravisEz13** [comment]
Reviewed 10 of 11 files at r4, 9 of 14 files at r5.
Review status: 23 of 29 files reviewed at latest revision, 5 unresolved discussions.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/powershell/xwebadministration/141)*

**Comment by TravisEz13** [comment]
Reviewed 1 of 11 files at r4, 5 of 14 files at r5.
Review status: all files reviewed at latest revision, 6 unresolved discussions.

---

*[DSCResources/MSFT_xIisFeatureDelegation/MSFT_xIisFeatureDelegation.psm1, line 18 \[r5\]](https://reviewable.io:443/reviews/powershell/xwebadministration/141#-KJqYsLafdap9t4s3Qx0:-KJqYsLafdap9t4s3Qx1:478656434) ([raw file](https://github.com/powershell/xwebadministration/blob/27668a47ec8bc91cae629678032aca2553acd164/DSCResources/MSFT_xIisFeatureDelegation/MSFT_xIisFeatureDelegation.psm1#L18)):*
> ```PowerShell
> 
> <#
>     The Get-TargetResource cmdlet.
> ```

why are we removing these comments?

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/powershell/xwebadministration/141)*

**Comment by mbreakey3** [comment]
*[DSCResources/MSFT_xIisFeatureDelegation/MSFT_xIisFeatureDelegation.psm1, line 18 \[r5\]](https://reviewable.io:443/reviews/powershell/xwebadministration/141#-KJqYsLafdap9t4s3Qx0:-KJqrEEJGtq50wv7XWWK:1903624362) ([raw file](https://github.com/powershell/xwebadministration/blob/27668a47ec8bc91cae629678032aca2553acd164/DSCResources/MSFT_xIisFeatureDelegation/MSFT_xIisFeatureDelegation.psm1#L18)):*
<details><summary><i>Previously, TravisEz13 (Travis Plunk) wrote…</i></summary>
> why are we removing these comments?
</details>
They aren't in the help-comment style that we want, but I will add them back in the correct format

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/powershell/xwebadministration/141)*

**Comment by TravisEz13** [comment]
Reviewed 2 of 2 files at r6.
Review status: all files reviewed at latest revision, 5 unresolved discussions.

---

*[DSCResources/MSFT_xIisModule/xIisModuleDesigner.ps1, line 32 \[r2\]](https://reviewable.io:443/reviews/powershell/xwebadministration/141#-KJgWkJxPkQjFXr6JXE2-r2-32:-KJrE6F2fklYSopN2vgo:-462437519) ([raw file](https://github.com/powershell/xwebadministration/blob/adc360956b7ffc684b42953280821ad73b53de94/DSCResources/MSFT_xIisModule/xIisModuleDesigner.ps1#L32)):*
<details><summary><i>Previously, PlagueHO (Daniel Scott-Raynsford) wrote…</i></summary>
> I know not strictly necessary and all, but could we wrap the updating... in quotes - just sort of feels wrong :smile:
</details>
lgtm

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/powershell/xwebadministration/141)*

**Comment by mbreakey3** [comment]
@PlagueHO - any further comments before I merge these changes?

**Comment by PlagueHO** [comment]
:lgtm: 

<details><summary><i>Previously, mbreakey3 (Mariah) wrote…</i></summary>
> @PlagueHO - any further comments before I merge these changes?
</details>


---

Review status: all files reviewed at latest revision, 4 unresolved discussions.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/powershell/xwebadministration/141#-:-KJt4WNPaVewNQpsvyno:980077423)*

**Pull request merged**